RA_Integration.sln

```diff
@@ -12,35 +12,72 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "RA_Integration.Tests", "tests\RA_Integration.Tests.vcxproj", "{876DEBDF-C7C8-4713-8CEF-EBDEE65306A7}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "RA_Integration.Data.rcheevos", "rcheevos_lib\RA_Integration.Data.rcheevos.vcxproj", "{9D55EBE7-1392-4FA1-A9B9-F022F764CE35}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Analysis|Win32 = Analysis|Win32
+		Analysis|x64 = Analysis|x64
```

Comment:
These aren't configured, should we remove these manually?

Reply:
How did they get in there in the first place? If you add them for any project, they will infect the other projects as well. This project probably doesn't need Analysis configurations either.

Reply:
I think they got added when I added the rcheevos project. Those configs are for the the whole solution, the rcheevos project doesn't have an analysis config.